Extension/package.json

```diff
@@ -6458,6 +6459,18 @@
                     "description": "%c_cpp.codeActions.refactor.extract.function.description%"
                 }
             }
+        ],
+        "languageModelTools": [
+            {
+                "id": "cpptools-lmtool-configuration",
+                "name": "cpp",
+                "displayName": "C/C++ configuration",
+                "canBeInvokedManually": true,
+                "userDescription": "Configuration of the active C or C++ file, like language standard version and target platform.",
```

Comment:
Not sure if it's supported here (yet?), but ultimately this should get localized.  I assume that would be similar to other `"%<desc>%"` loc strings supported in some other package.json fields.

Reply:
Updated to use strings from `package.nls.json` and it appears to work.